cunumeric/random/bitgenerator.py

```diff
@@ -33,21 +33,43 @@ class BitGenerator:
     def __init__(
         self,
         seed: Union[int, None] = None,
-        generatorType: BitGeneratorType = BitGeneratorType.DEFAULT,
         forceBuild: bool = False,
```

Comment:
Should `forceBuild` be documented, or is it an internal thing?

Reply:
That parameter will be removed in a follow-on PR. I don't want to modify the functions while I'm adding a whole bunch of docstrings.